test/io/wkt/wkt.cpp

```diff
@@ -76,6 +106,44 @@ void test_wkt(std::string const& wkt, std::string const& expected,
     check_wkt(boost::variant<G>(geometry), expected);
 }
 
+template <typename G>
+void test_wkt_to_from(std::string const& wkt, std::string const& expected,
+              std::size_t n, double len = 0, double ar = 0, double peri = 0)
+{
+    G geometry;
+
+    geometry = bg::from_wkt<G>(wkt);
+
+    /*
+    std::cout << "n=" << bg::num_points(geometry)
```

Comment:
you could use  `#ifdef BOOST_GEOMETRY_TEST_DEBUG` here instead of commenting out

Reply:
@vissarion I will do that, thanks for the suggestions.